frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -104,6 +104,7 @@ export const ExpressionEditorSuggestions = forwardRef<
       radius="xs"
       withinPortal
       middlewares={{
+        size: true,
```

Comment:
In our custom override we always applied the `size` middleware. To have the same behavior, we have to enable it everywhere.